src/current/v23.1/migration-strategy-lift-and-shift.md

```diff
@@ -0,0 +1,142 @@
+---
+title: "Migration Strategy: Lift and Shift"
+summary: Learn about the 'Lift and Shift' data migration strategy
+toc: true
+docs_area: migrate
+---
+
+There are multiple strategies for [migrating off legacy technology](migration-overview.html#molt) to CockroachDB.
+
+This page discusses the 'Lift and Shift' strategy for migrating your data, which is the most commonly used approach. This may not be the right approach if a strong application service continuity during the migration is required. However, it may be a viable method if downtime is permitted. It works particularly well when the application accurately maintains [primary key](primary-key.html) and [foreign key](foreign-key.html) constraints.
```

Comment:
I think we should slightly hedge the most commonly used to something like "which is a popular approach"

Should we say application downtime?

Also, why does it work well for PK and FK constraints?

Reply:
> I think we should slightly hedge the most commonly used to something like "which is a popular approach"

updated to say "a commonly used approach"

> Should we say application downtime?

updated mentions of "downtime" to clarify "application downtime"

> Also, why does it work well for PK and FK constraints?

this was based on some internal writings. we could probably get into it for a v2 but for now I've deleted the PK / FK bit based on some feedback via https://cockroachlabs.atlassian.net/wiki/x/IgAgtw from @sheaffej and @a-entin